kimhaejung/routes/postRouter.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const postController = require("../controllers/postController");
+const validateToken = require("../utils/auth");
+
+const router = express.Router();
+
+router.post("", validateToken, postController.posting);
+router.get("", postController.getAllPosts);
+router.get("/users/:userId", postController.getIdPosts);
```

Comment:
userId는 path parameter가 아닌, token을 통해 얻어 주세요.
```suggestion
router.get("/users", postController.getIdPosts);
```